www/artists/mbolen/index.html

```diff
@@ -76,7 +76,7 @@ <h1 class="c4">
                         <div class="card-item c7">
                             <p class="c6">
                                 Gallery
-                            </p><img class="img-card" src="./img/wife.jpg" id="slide" alt="no alt text">
+                            </p><img class="img-card" src="./img/wife.jpg" id="slide" alt="no alt text"</img>  <img class="img-card" src="./img/bol2.jpg" id="slide" alt="no alt text"</img><img class="smallercallout img-card" src="./img/callout.jpg" id="slide" alt="no alt text"</img>
```

Comment:
img _should_ be a self-closing tag. You shouldn't need the </img>s. The only reason it wasn't working was because you didn't close the tag at all - you didn't have a ">" before the second image.